shadowsocks-rust.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: shadowsocks-rust
-  version: 1.20.0
+  version: 1.20.1
```

Comment:
Is this related / intentional?

Reply:
I wanted to get CI passing, the check update thing was mad at me and I don't have superpowers to merge.

Reply:
@jonjohnsonjr update check is not required for merges, one can merge things as is. Likely can bump this separately, but it's also ok to do it here.